src/tree/transformer.ts

```diff
@@ -213,11 +213,11 @@ const getMetadata = (element: Source): ComponentMetadata => {
 };
 
 const getComponentDirectives = (metadata: ComponentMetadata): Array<string> => {
-  if (metadata == null || metadata.directives == null) {
+  if (metadata == null || metadata['directives'] == null) {
     return [];
```

Comment:
I was wondering whether we wanted to retain any RC4/deprecated RC5 support for this; but Igor clarifies that it's a non-goal.

I'm stubbing this out, we will need to implement this a different way in the future.